src/components/molecules/TickerCard.tsx

```diff
@@ -26,9 +34,11 @@ const TickerCard = ({ price, name, ticker }: Props) => {
         {ticker}
       </a>
       <ButtonsContainer>
+        <Button onClick={handleOpen} variant="outlined" size="small">More info</Button>
         <Button variant="contained" size="small" color="info" onClick={() => handleBuyOrder(ticker)}>BUY $1 {name}</Button>
         <Button variant="contained" size="small" color="success" onClick={() => handleSellOrder(ticker)}>SELL $1 {name}</Button>
       </ButtonsContainer>
+      <ModalDialog isOpen={isModalOpen} handleClose={handleClose} {...tickerInfo}/>
```

Comment:
...and since we did not need tickerInfo as constant, we can spread Props here instead of tickerInfo.